src/main/java/com/yahoo/bullet/querying/Querier.java

```diff
@@ -673,4 +691,14 @@ private void incrementRate() {
     private String getMetaKey() {
         return metaKeys.getOrDefault(Meta.Concept.QUERY_METADATA.getName(), null);
     }
+
+    private Clip postAggregation(Clip clip) {
+        Clip result = clip;
```

Comment:
Why reassign to "result"?  Just use "clip"?

Reply:
It's just my habit. I don't want to reassign a value to the parameter variables unless it has to. Yes, for here I can change to use clip directly.

Reply:
Yea removing the unnecessary reassignment would be good, but it's very minor.  I will approve now.  :+1: